test/v1/mocks/connectors/MockListBsasConnector.scala

```diff
@@ -31,10 +31,10 @@ trait MockListBsasConnector extends MockFactory {
 
   object MockListBsasConnector {
 
-    def listBsas(requestData: ListBsasRequest): CallHandler3[ListBsasRequest, HeaderCarrier, ExecutionContext, Future[DesOutcome[ListBsasResponse[BsasEntries]]]] = {
+    def listBsas(requestData: ListBsasRequest): CallHandler4[ListBsasRequest, HeaderCarrier, ExecutionContext, String, Future[DesOutcome[ListBsasResponse[BsasEntries]]]] = {
```

Comment:
This line can be be shortened using the standard CallHandler i.e. **: CallHandler[Future[DesOutcome[ListBsasResponse[BsasEntries]]]]** instead of using CallHandler4